src/primitives/roundedRectangle.js

```diff
@@ -0,0 +1,57 @@
+const {defaultResolution2D, EPS} = require('../core/constants')
+
+const vec2 = require('../math/vec2')
+
+const {geom2} = require('../geometry')
+
+/** Construct a rounded rectangle.
+ * @param {Object} [options] - options for construction
+ * @param {Array} [options.center=[0,0]] - center of rounded rectangle
+ * @param {Array} [options.radius=[1,1]] - radius of rounded rectangle, width and height
+ * @param {Number} [options.roundradius=0.2] - round radius of corners
+ * @param {Number} [options.resolution=defaultResolution2D] - number of sides per 360 rotation
+ * @returns {geom2} new 2D geometry
+ *
+ * @example
+ * let myrectangle = roundedRectangle({radius: [5, 10], roundradius: 2})
+ */
+const roundedRectangle = (options) => {
+  const defaults = {
+    center: [0, 0],
+    radius: [1, 1],
```

Comment:
same as for rectangle : size instead of radius

Reply:
I'm not oppose to renaming 'radius' to 'size' but just to be sure...

radius: [x, y, z] to size: [x, y, z]
radius: N to size: N

Examples:
- circle({**size: 5,** center: [4, 3]}}
- roundedCube({**size: [5, 10, 10]**, segments: 12})

@kaosat-dev Still fine?

Reply:
What about angles... do switch to radians?

Reply:
err wait : 
- radius/diameter is for circles, or anything based on that, because that is their mathematical definition
- size is for other type of dimentions
do not over think this :) we do not want the same parameters for all shapes, only where that makes sense !
angles in radians: yes that was the understanding we came to before right ?

Reply:
Renamed radius to size for cuboid, cube, rectangle, square, roundedCube, roundedRectangle. That completes the NAME change, but the meaning of size is still 'radius'.

Do we want size to be the full size (100% width, depth, height)?
I think full 100% size makes more sense.

Back to you @kaosat-dev . What do you think?

Reply:
size is a single 2d/3d array, I am fine with that to be honest, I think having separate width, length, height parameters might be overkill (and it would prevent passing 2d/3d arrays as inputs directly)

Reply:
I still need to adjust angles to radians for...
- arc
- cylinder / cylinderElliptic
- star
- torus ('roti' option, rotation angle of base circle) rotate a circle? does that make sense?

Reply:
thanks! awesome ! 
hmm it does seem odd like that for torus, it makes sense visually though ?